test/e2e/testkit/otlp/traces/traces.go

```diff
@@ -49,3 +53,26 @@ func MakeTraces(traceID pcommon.TraceID, spanIDs []pcommon.SpanID, attributes pc
 
 	return traces
 }
+
+func MakeDataSender(otlpPushURL string) (testbed.TraceDataSender, error) {
```

Comment:
I suggest renaming this constructor function to `NewDataSender`. I consistently sticked to this naming pattern inside the `testkit` package.

Reply:
Agree